packages/hoppscotch-common/src/components/http/authorization/ApiKey.vue

```diff
@@ -28,7 +28,13 @@
       >
         <HoppSmartSelectWrapper>
           <HoppButtonSecondary
-            :label="auth.addTo || t('state.none')"
+            :label="
+              auth.addTo
+                ? auth.addTo === 'HEADERS'
+                  ? t('authorization.pass_by_headers_label')
```

Comment:
The i18n strings used are not defined. 

<img width="361" alt="image" src="https://github.com/hoppscotch/hoppscotch/assets/70131076/76f2dc56-212c-4b2b-8158-c88f56e1821c">

Reply:
i missed to checkin that. my bad. thank you for pointing out.